gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -279,56 +249,98 @@ MM_VerboseHandlerOutput::handleInitialized(J9HookInterface** hook, uintptr_t eve
 #endif /* defined(S390) || defined(J9ZOS390) */
 	}
 #endif /* OMR_GC_CONCURRENT_SCAVENGER */
-	writer->formatAndOutput(env, 1, "<attribute name=\"maxHeapSize\" value=\"0x%zx\" />", event->maxHeapSize);
-	writer->formatAndOutput(env, 1, "<attribute name=\"initialHeapSize\" value=\"0x%zx\" />", event->initialHeapSize);
+
+	buffer->formatAndOutput(env, 1, "<attribute name=\"maxHeapSize\" value=\"0x%zx\" />", _extensions->memoryMax);
+	buffer->formatAndOutput(env, 1, "<attribute name=\"initialHeapSize\" value=\"0x%zx\" />", _extensions->initialMemorySize);
+
 #if defined(OMR_GC_COMPRESSED_POINTERS)
+	MM_GCExtensions *extensionsExt = MM_GCExtensions::getExtensions(env);
 	if (env->compressObjectReferences()) {
-		writer->formatAndOutput(env, 1, "<attribute name=\"compressedRefs\" value=\"true\" />");
-		writer->formatAndOutput(env, 1, "<attribute name=\"compressedRefsDisplacement\" value=\"0x%zx\" />", 0);
-		writer->formatAndOutput(env, 1, "<attribute name=\"compressedRefsShift\" value=\"0x%zx\" />", event->compressedPointersShift);
+		buffer->formatAndOutput(env, 1, "<attribute name=\"compressedRefs\" value=\"true\" />");
+		buffer->formatAndOutput(env, 1, "<attribute name=\"compressedRefsDisplacement\" value=\"0x%zx\" />", 0);
+		buffer->formatAndOutput(env, 1, "<attribute name=\"compressedRefsShift\" value=\"0x%zx\" />", extensionsExt->accessBarrier->compressedPointersShift());
```

Comment:
CR shift is the only thing from accessBarrier. We can get it from omrVM->_compressedPointersShift, and thus avoid accessing accessBarrier and MM_GCExtensions that are OpenJ9 specific